src/main/java/com/example/garage_sql/repository/CarRepositoryImp.java

```diff
@@ -0,0 +1,79 @@
+package com.example.garage_sql.repository;
+
+import com.example.garage_sql.model.Car;
+import com.example.garage_sql.repository.Dao.CarRepository;
+import com.example.garage_sql.repository.mapper.CarMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class CarRepositoryImp implements CarRepository {
+    private final JdbcTemplate jdbcTemplate;
+    List<Car> cars = new ArrayList<>();
```

Comment:
bad idea, this class-level variable will be representing a state, and you want your service to be stateless